### ISSUE TITLE
Test loss fluctuating around train. 

### ISSUE THREAD

**Issue by pankajkalania** [keras-team/keras#10558]
Below code is for my LSTM model.  My data has 5 features. With 4 of them being input feature and final one as output. Can anyone help me in fine tuning the model ? 

```
`def model(filename):
    r=split_data(filename)
    n_days = 10
    n_features = 5
    train_X,train_y,test_X,test_y,scaler=r[0],r[1],r[2],r[3],r[4]
    print train_X.shape,train_y.shape,test_X.shape,test_y.shape
    model = Sequential()
    model.add(LSTM(50,input_shape=(train_X.shape[1], train_X.shape[2]))) 
    model.add(Dense(1,activation='sigmoid'))
    #sgd = optimizers.SGD(lr=0.01, decay=1e-6, momentum=0.9, nesterov=True)
    adam=optimizers.Adam(lr=0.0000001, beta_1=0.9, beta_2=0.999, epsilon=0.00000001, decay=0.0, amsgrad=False)
    model.compile(loss='mae', optimizer='adam')
    #model.compile(optimizer='rmsprop', loss='categorical_crossentropy', metrics=['accuracy'])
    # fit network
    history = model.fit(train_X, train_y, epochs=200, batch_size=100, validation_data=(test_X, test_y), verbose=2, shuffle=False)
    model.summary()
    # plot history
    pyplot.plot(history.history['loss'], label='train')
    pyplot.plot(history.history['val_loss'], label='test')
    pyplot.legend()
    pyplot.show()
    # make a prediction
    yhat = model.predict(test_X)
    test_X = test_X.reshape((test_X.shape[0], n_days*n_features))
    # invert scaling for forecast
    inv_yhat = concatenate(( test_X[:, -5:-1],yhat), axis=1)
    inv_yhat = scaler.inverse_transform(inv_yhat)
    inv_yhat = inv_yhat[:,-1]
    # invert scaling for actual
    test_y = test_y.reshape((len(test_y), 1))
    inv_y = concatenate((test_y, test_X[:, -5:-1]), axis=1)
    inv_y = scaler.inverse_transform(inv_y)
    inv_y = inv_y[:,-1]
    # calculate RMSE
    rmse = sqrt(mean_squared_error(inv_y, inv_yhat))
    print('Test RMSE: %.3f' % rmse)`
```

See test and train loss graph [here](http://bit.ly/2lCK8op).
